### PR TITLE
Add HTTP breadcrumbs for Dart and Flutter

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

File: src/platforms/flutter/usage/advanced-usage.mdx [hunk 1]
```diff
@@ -28,6 +28,8 @@ MaterialApp(
 )
 ```
 
+To track automatic Breadcrumbs for `HTTP` requests, Check out the [SentryHttpClient]((/platforms/dart/usage/advanced-usage/#automatic-breadcrumbs)) Wrapper for the [http](https://pub.dev/packages/http) library.
```
Comment (marandaneto) [inline src/platforms/flutter/usage/advanced-usage.mdx:31]:
```suggestion
To track automatic Breadcrumbs for `HTTP` requests, Check out the [SentryHttpClient](/platforms/dart/usage/advanced-usage/#automatic-breadcrumbs) Wrapper for the [http](https://pub.dev/packages/http) library.
```

File: src/wizard/dart/index.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Get the SDK from [pub.dev](https://pub.dev/packages/sentry) by adding the follow
 
 ```yml {filename:pubspec.yaml}
 dependencies:
-  sentry: ^4.0.0-beta.1
+  sentry: ^4.0.0-beta.2
```
Comment (marandaneto) [inline src/wizard/dart/index.md:12]:
```suggestion
  sentry: ^4.0.0-beta.1
```

File: src/wizard/flutter/index.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ Get the SDK from [pub.dev](https://pub.dev/packages/sentry_flutter) by adding th
 
 ```yml {filename:pubspec.yaml}
 dependencies:
-  sentry_flutter: ^4.0.0-beta.1
+  sentry_flutter: ^4.0.0-beta.2
```
Comment (marandaneto) [inline src/wizard/flutter/index.md:12]:
```suggestion
  sentry_flutter: ^4.0.0-beta.1
```

File: src/platforms/flutter/usage/advanced-usage.mdx [hunk 4]
```diff
@@ -28,6 +28,8 @@ MaterialApp(
 )
 ```
 
+To track automatic Breadcrumbs for `HTTP` requests, Check out the [SentryHttpClient](/platforms/dart/usage/advanced-usage/#automatic-breadcrumbs) Wrapper for the [http](https://pub.dev/packages/http) library.
```
Comment (philipphofmann) [inline src/platforms/flutter/usage/advanced-usage.mdx:31]:
```suggestion
To track automatic breadcrumbs for `HTTP` requests, check out the [SentryHttpClient](/platforms/dart/usage/advanced-usage/#automatic-breadcrumbs) wrapper for the [http](https://pub.dev/packages/http) library.
```

**Review by bruno-garcia — APPROVED** [review]
Lgtm

File: src/platforms/dart/usage/advanced-usage.mdx [hunk 5]
```diff
@@ -7,6 +7,51 @@ sidebar_order: 2
...
+    'name': 'doodle',
+    'color': 'blue',
+  });
+  print('Response body: ${response.body}');
+} finally {
+  client.close();
+}
+```
+
+Or the `SentryHttpClient` can also be used as a wrapper for your own `HTTP Client`:
```
Comment (philipphofmann) [inline src/platforms/dart/usage/advanced-usage.mdx:35]:
```suggestion
The `SentryHttpClient` can also be used as a wrapper for your own `HTTP Client`:
```

File: src/platforms/dart/usage/advanced-usage.mdx [hunk 6]
```diff
@@ -7,6 +7,51 @@ sidebar_order: 2
 
 - For the usage of the Dart SDK, the minimal required Dart SDK version is `2.8.0` and Flutter SDK version is `1.17.0`
 
+## Automatic Breadcrumbs
+
+If you want to track automatic [Breadcrumbs for HTTP requests](https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#breadcrumb-types), you can add a Sentry wrapper that does it automatically for you.
```
Comment (philipphofmann) [inline src/platforms/dart/usage/advanced-usage.mdx:12]:
```suggestion
To track automatic [breadcrumbs for HTTP requests](https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#breadcrumb-types), you can add a Sentry wrapper that does this automatically.
```

**Review by philipphofmann — APPROVED** [review]
Just minor suggestions for the wording.

File: src/platforms/dart/usage/advanced-usage.mdx (hunk 6 above)
Comment (marandaneto) [reply src/platforms/dart/usage/advanced-usage.mdx:12]: I'd rather keep `Breadcrumbs` and `it`, the rest makes sense to me

File: src/platforms/flutter/usage/advanced-usage.mdx (hunk 4 above)
Comment (marandaneto) [reply src/platforms/flutter/usage/advanced-usage.mdx:31]: I'd rather keep it with a capital letter, our docs mention always `B`readcrumbs
https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#breadcrumb-types

**Comment by philipphofmann** [comment]
No strong feelings on the suggestions @marandaneto. Do as you wish.

**Pull request merged**